pyproject.toml

```diff
@@ -100,77 +100,55 @@ line-length = 88
 
 [tool.ruff]
 line-length = 88
+target-version = "py39"
```

Comment:
```suggestion
```

afaik this is not needed and ruff will derive it from `requires-python`.

Reply:
Good catch, that's indeed the case: https://docs.astral.sh/ruff/settings/#target-version